core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-layout/edit-ema-layout.component.ts

```diff
@@ -138,17 +142,17 @@ export class EditEmaLayoutComponent implements OnInit, OnDestroy {
      * Handle the success save template
      *
      * @private
-     * @param {DotPageRender} _
+     * @template T
+     * @param {T=unkonwm} page // To avoid getting type error with DotPageRender and DotPageApiResponse
      * @memberof EditEmaLayoutComponent
      */
-    private handleSuccessSaveTemplate(page: DotPageRender): void {
+    private handleSuccessSaveTemplate<T = unknown>(page: T): void {
```

Comment:
Why Generic? You use `as DotPageApiResponse` below

Reply:
The `DotPageLayout` service returns a `DotPageRender`, the same as `DotPageApiResponse` but a type mismatch exists. Don't want to start fixing types here, this should be merged ASAP

Reply:
I think you can do `T extends DotPageApiResponse`. Try it to see if it works, it's more readable that way

Reply:
Sadly, it's not working! We should create a GD ticket to unify the Page API Types Response